### ISSUE TITLE
javascript:S2819 - false positive when variable used

### ISSUE THREAD

**Issue by lukeapage** [SonarSource/SonarJS#1278]
**I want to report a bug.**

**SonarJS version:** 5.0.0.6962
**SonarQube version:** 7.6

**Rule key:** javascript:S2819

**Reproducer** 
```
this.foreignWindow.postMessage(message, this.targetOrigin);
```

**Expected behavior**

this.targetOrigin is not "*" and the author has thought about restricting postMessage to the domain - this rule should not error on this line of code

**Comment by eric-therond-sonarsource** [comment]
Ticket to update this rule https://github.com/SonarSource/SonarJS/issues/1772
